torch/csrc/jit/script/compiler.cpp

```diff
@@ -474,7 +474,7 @@ static Value* materializeConstant(
   }
 
   WithInsertPoint guard(graph.block()->nodes().front());
-  auto new_constant = graph.insertConstant(val, r);
+  auto new_constant = graph.insertConstant(val, nullptr, r);
```

Comment:
It's weird to put a default value in the `insertConstant` but then not propagate it to `Graph::insertConstant`

Reply:
This is only called with ivalues that the type can be deduced form. 
I guess i could make it the following instead ?
```
template <class T, class Type>
static Value* materializeConstant
```

Reply:
Oh, nevermind -- I didn't see that these also have the range argument and prevent using the default argument for type. This should be fine.